core/src/test/scala/org/broadinstitute/dsde/rawls/webservice/ApiServiceSpec.scala

```diff
@@ -414,16 +445,19 @@ trait ApiServiceSpec
     val sealedInstrumentedRoutes: Route = captureRequestMetrics {
       traceRequests { otelContext =>
         sealRoute(
-          adminRoutes(otelContext) ~
```

Comment:
The route order is potentially significant.
These were rearranged so they were the same order as in `RawlsApiService`.